roboslack-api/src/main/java/com/palantir/roboslack/api/attachments/Attachment.java

```diff
@@ -72,7 +72,6 @@ public static Builder builder() {
     @Value.Check
     protected final void check() {
         checkArgument(!Strings.isNullOrEmpty(fallback()), "Attachment fallback message cannot be null or empty");
-        MorePreconditions.checkDoesNotContainMarkdown(FALLBACK_FIELD, fallback());
```

Comment:
Even if we had something like:
`Some text-with-dashes` we should still be able to handle that correctly as fallback text and not just reject the entire thing.

My thoughts here was that the user should know that whatever characters they add to this will not be processed as Markdown, instead of just rejecting it.